sympy/calculus/tests/test_util.py

```diff
@@ -57,6 +57,7 @@ def test_function_range():
         ) == Interval(-1, 1)
     assert function_range(cos(x), x, S.EmptySet) == S.EmptySet
     assert function_range(x/sqrt(x**2+1), x, S.Reals) == Interval.open(-1,1)
+    assert function_range(tan(x)**2 + tan(3*x)**2 + 1, x, S.Reals) == Interval(1,oo)
```

Comment:
This should be put in a separate test function marked `@slow` because it takes over 10 seconds.

Reply:
okay I will do that